src/care/ProfilePlugin.cpp

```diff
@@ -0,0 +1,54 @@
+#include "care/ProfilePlugin.h"
+#include "care/PluginData.h"
+
+/* CUDA profiling macros */
+#if defined(__CUDACC__) && CARE_HAVE_NVTOOLSEXT
+#include "nvToolsExt.h"
+#endif
+
+namespace care{
+   unsigned int ProfilePlugin::s_current_color = 0;
+   uint32_t ProfilePlugin::s_colors[7] = { 0x0000ff00, 0x000000ff, 0x00ffff00, 0x00ff00ff, 0x0000ffff, 0x00ff0000, 0x00ffffff };
```

Comment:
Comments since many people don't know whether 0x000000ff is purple or orange or whatever

Reply:
Or just add a TODO comment saying that we need label the colors.